libcloud/test/storage/fixtures/azure_blobs/list_containers_1.xml

```diff
@@ -1,10 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<EnumerationResults AccountName="https://account.blob.core.windows.net/">
+<EnumerationResults ServiceEndpoint="https://account.blob.core.windows.net/">
     <MaxResults>2</MaxResults>
     <Containers>
         <Container>
             <Name>container1</Name>
```

Comment:
any chance of a collision or orphaned results here? Maybe a random container name?

Reply:
This is a frozen/recorded API response not a live one, so no risk of collision. The libcloud tests don't make any live service calls (for that we have [c-w/libcloud-tests](https://github.com/c-w/libcloud-tests)).